content/en/storage-providers/seal-workers/post-workers.md

```diff
@@ -0,0 +1,155 @@
+
+{{< alert icon="tip" >}}
+When fetching parameter files, remember to set the [`IPFS_GATEWAY` variable when running from China]({{< relref "nodes-in-china" >}})
+{{< /alert >}}
+
+### Run the PoSt worker
+
+```shell
+lotus-worker run <flags>
+```
+
+The above command will start the worker. You´ll need to specify which PoSt operation you want the worker to perform with one of the following flags set to true.
+
+```
+   --winningpost                 enable winning post (default: false)
+   --windowpost                  enable window post (default: false)
+```
+
+A PoSt worker instance can only be either a winningPoSt worker, or a windowPoSt worker.
+
+When a winningPoSt or windowPoSt worker connects to the _lotus-miner_, the lotus miner will delegate all winningPoSt or windowPoSt tasks to that worker. If both tasks are delegated to seperate PoSt workers, no PoSt tasks will be executed locally on the miner instance. If a worker is stopped, the lotus-miner instance switches back to local PoSt automatically.
+
+You can verify that PoSt workers are connected to the lotus-miner with `lotus-miner proving workers`:
+
+```shell
+$ lotus-miner proving workers
+Worker 0, host windowPoSt
+        CPU:  [                                                                ] 0/48 core(s) in use
+        RAM:  [|                                                               ] 1% 4.005 GiB/119.2 GiB
+        VMEM: [|                                                               ] 1% 4.005 GiB/119.2 GiB
+        GPU:  [                                                                ] 0% 0.00/1 gpu(s) in use
+        GPU: GeForce RTX 2080 Ti, not used
+
+Worker 1, host winningPoSt
+        CPU:  [                                                                ] 0/16 core(s) in use
+        RAM:  [|                                                               ] 1% 4.005 GiB/119.2 GiB
+        VMEM: [|                                                               ] 1% 4.005 GiB/119.2 GiB
+        GPU:  [                                                                ] 0% 0.00/1 gpu(s) in use
+        GPU: GeForce RTX 2080 Ti, not used
+```
+
+You can also verify that PoSt workers are connected in the workers-section of `lotus-miner info`.
+
+```shell
+$ lotus-miner info
+Workers:
+	Seal: 15
+	WdPoSt: 4
+	WinPoSt: 3
+```
+
+### Advanced settings
+
+Although the default settings are reasonable you can configure some advanced settings when running the PoSt workers, that can be tested for local optimizations of your hardware.
+
+{{< alert icon="tip" >}}
+Use with caution as changing these values to extremes might cause you to miss windowPoSt.
+{{< /alert >}}
+
+```
+   --post-parallel-reads value   maximum number of parallel challenge reads (0 = no limit) (default: 128)
```

Comment:
```suggestion
   --post-parallel-reads value   # maximum number of parallel challenge reads (0 = no limit) (default: 128)
```

Reply:
I think maybe we should keep them without the #, at least that is what we have done here: [https://lotus.filecoin.io/storage-providers/seal-workers/seal-workers/#run-the-worker](https://lotus.filecoin.io/storage-providers/seal-workers/seal-workers/#run-the-worker)